containers/podman-desktop
# chore: hide tooltip when opening context/dropdown/buddy widget menu

Signed-off-by: Sonia Sandler <ssandler@redhat.com>

### What does this PR do?
When right-clicking on a nav item in the left side navigation bar, the tooltip is sometimes still visible when the context menu opens. This PR closes the tooltip when a context menu of a nav item opens. However, this does not include the buddy icon tooltip since there is a PR in progress for its menu.  

### Screenshot / video of UI
I'm working on Linux and I don't have the problem mentioned in https://github.com/containers/podman-desktop/issues/8335, so I didn't have any visual changes/updates on my computer for the NavItems

Dropdown/accounts menu example:

[Screencast from 2024-08-29 08-11-46.webm](https://github.com/user-attachments/assets/d9db02c4-ce7d-4acb-a70c-fe82b5e62185)



### What issues does this PR fix or reference?
Closes https://github.com/containers/podman-desktop/issues/8335



### How to test this PR?
Right click any of the nav items on the left navigation bar and check that the tooltip doesn't show
